common/government_types/00_babylon_governments.txt

```diff
@@ -0,0 +1,126 @@
+	transfer_of_power = presidential_elective
+	new_leader_on_reform_government = no
+
+	male_ruler = "RULER_TITLE_VICEROY"
+	female_ruler = "RULER_TITLE_VICEROY"
+	
+	possible = {
+		has_law = law_type:law_presidential_republic
+
+		OR = {
+			AND = {
+				exists = c:CHI
+				is_subject_of = c:CHI
+				any_primary_culture = {
+					OR = {
+						has_discrimination_trait = han_chinese
+						has_discrimination_trait = sinosphere
+						has_discrimination_trait = central_asian_heritage
+					}
+				}
+				NOT = {
+					c:LAN = ROOT
+				}
+			}
+				
+			AND = {
+				exists = c:TPG
+				is_subject_of = c:TPG
+				any_primary_culture = {
+					OR = {
+						has_discrimination_trait = han_chinese
+						has_discrimination_trait = sinosphere
+						has_discrimination_trait = central_asian_heritage
+					}
+				}
+				NOT = {
+					c:LAN = ROOT
+				}
+			}
+		}
+	}
+	
+	on_government_type_change = {
+		change_to_presidential_elective = yes
+	}
+	on_post_government_type_change = {
+		post_change_to_presidential_elective = yes
+	}
+}
+
+gov_bulgarian_vilayet = {								
+	transfer_of_power = hereditary
+	
+	male_ruler = "RULER_TITLE_GOVERNOR"			# Governor. This post would usually be appointed by the Ottoman Sultan, and not be hereditary. 
+	female_ruler = "" # Empty on purpose
+
+	male_heir = "RULER_TITLE_VICE_GOVERNOR"	# Governor of a Vilayet's subunit, the Sanjak, who would assume command in the Vali's absence
+	female_heir = "" # Empty on purpose
+	
+	possible = {
+		has_law = law_type:law_monarchy
+		exists = c:BUL
+		exists = c:TUR
+		c:BUL = ROOT
+		is_subject_of = c:TUR		
+	}		
+
+	on_government_type_change = {
+		change_to_hereditary = yes
+	}
+	on_post_government_type_change = {
+		post_change_to_hereditary = yes
+	}
+}
+
+gov_transcaucasian_governorate = {								
+	transfer_of_power = hereditary
+	
+	male_ruler = "RULER_TITLE_GOVERNOR_GENERAL"			# Governor. This post would usually be appointed by the Ottoman Sultan, and not be hereditary. 
+	female_ruler = "" # Empty on purpose
+
+	male_heir = "RULER_TITLE_VICE_GOVERNOR"	# Governor of a Vilayet's subunit, the Sanjak, who would assume command in the Vali's absence
+	female_heir = "" # Empty on purpose
+	
+	possible = {
+		has_law = law_type:law_monarchy
+		exists = c:GEO
+		exists = c:RUS
+		c:GEO = ROOT
+		is_subject_of = c:RUS		
+	}		
+
+	on_government_type_change = {
+		change_to_hereditary = yes
+	}
+	on_post_government_type_change = {
+		post_change_to_hereditary = yes
+	}
+}
+
+gov_bulgarian_tsardom = {
+	transfer_of_power = hereditary
+	
+	male_ruler = "RULER_TITLE_TSAR" 
+	female_ruler = "RULER_TITLE_TSARINA"
+
+	male_heir = "RULER_TITLE_TSESAREVICH"
+	female_heir = "RULER_TITLE_TSAREVNA" 
+	
+	possible = {
+		has_law = law_type:law_monarchy
+		exists = c:BUL
+		c:BUL = ROOT
+		
+		has_variable = var_bulgarian_independence
+		
+	}
+
+	on_government_type_change = {
+		change_to_hereditary = yes
+	}
+	on_post_government_type_change = {
+		post_change_to_hereditary = yes
+	}
+}
```

Comment:
was this also not defined in the tanzimat? lol. if it is defined then this is duplication